src/components/layout.js

```diff
@@ -25,6 +28,7 @@ const Layout = ({ children }) => {
   return (
     <>
       <Header siteTitle={data.site.siteMetadata.title} />
+      <NavBar navbarState={navbarOpen} onClick={() => handleNavbar} />
```

Comment:
Just make this `onClick={handleNavbar}`. If you do it with this arrow function way, it would anyway need to be `() => handleNavbar()`. But first way is better anyway.